backend/uclapi/dashboard/models.py

```diff
@@ -45,6 +47,24 @@ class Meta:
         _DATABASE = 'default'
 
 
+class TemporaryToken(models.Model):
+    id = models.CharField(
```

Comment:
@wilhelmklopp this is the only thing that's left to be fixed. 
Django wasn't allowing me to run the migration after I updated it.